common/containers/Tabs/GenerateWallet/components/DownloadWallet.jsx

```diff
@@ -27,11 +27,16 @@ export default class DownloadWallet extends Component {
   }
 
   componentWillMount() {
```

Comment:
I tried something like
```
  async componentWillMount() {
    if (!this.props.wallet) return;
    const keystore = await this.props.wallet.toKeystore(this.props.password);
    this.setState({ keystore });
  }
```
which seem to work when using the app, but flow threw an error. Seems to be a flow limitation?

https://github.com/facebook/flow/issues/1803